razorpay-subscriptions.php

```diff
@@ -378,3 +345,30 @@ function razorpay_webhook_subscription_init()
 
     $rzpWebhook->process();
 }
+
+function razorpaySubscriptionPluginActivated()
+{
+    $rzp = new WC_Razorpay();
+    $api = $rzp->getRazorpayApiInstance();
+
+    $features = $api->request->request("GET", "accounts/me/features");
+
+    foreach ($features['assigned_features'] as $feature)
+    {
+        if($feature['name'] === 'subscriptions'
+            and $feature['entity_type'] === 'merchant')
+        {
+            update_option('rzp_subscription_webhook_enable_flag', true);
```

Comment:
@sonyy-m can we add a break statement after feature is found loop should not continue further

Reply:
added